strings/src/main/java/com/revenuecat/purchases/strings/ConfigureStrings.kt

```diff
@@ -0,0 +1,15 @@
+package com.revenuecat.purchases.strings
+
+object ConfigureStrings {
+    const val APP_BACKGROUNDED = Emojis.INFO + " App backgrounded"
+    const val APP_FOREGROUNDED = Emojis.INFO + " App foregrounded"
+    const val DEBUG_ENABLE = Emojis.INFO + " Debug logging enabled"
+    const val INVALID_CREDENTIALS_ERROR = Emojis.ROBOT + Emojis.ERROR + " There was a credentials issue. " +
+            "Check the underlying error for more details More info here: http://errors.rev.cat/play-service-credentials"
```

Comment:
There should probably be a `.` and a line break in between `details` and `More Info here:`

Reply:
that URL tho 😍